OntologyMapper/src/README.md

```diff
@@ -0,0 +1,253 @@
+
+| Field | Description |
+| --- | --- |
+| InputDtmi | The name of the input relationship |
+| OutputDtmi | The name of the output relationship |
+
+### PropertyProjections
+
+A collection of mappings from one property name to another property name. It is suggested that you only need to list mappings where the input property is not the same as the output property. Systems should assume that if a property is not listed in the mappings, that the output property will be the same as the input property.
+
+| Field | Description |
+| --- | --- |
+| OutputDtmiFilter | A regex filter which allows filtering so that the mapping is only applied to output property names which match the filter. * to apply to all output properties |
+| OutputPropertyName | The name of the output property to assign the input property to |
+| InputPropertyNames | A collection of names of the input property to assign to the output property. Can only specify multiple input properties if the output property is a collection |
+| IsOutputPropertyCollection | A flag which indicates whether or not the output property is a collection or not. This allows an input property which is not a collection to be assigned as an element in the output collection. |
+
+### FillProperties
+
+A collection of mappings from one or more property names to another property name so that if the output property is a required value and the input fields might be empty, that alternative fields can be used to populate the output field. It is suggested that you only need to list mappings where the input property is not the same as the output property. Systems should assume that if a property is not listed in the mappings, that the output property will be the same as the input property. Implementations should respect the order in which the input properties are specified so that the first non-empty valued input property is assigned to the output property
+
+| Field | Description |
+| --- | --- |
+| OutputDtmiFilter | A regex filter which allows filtering so that the mapping is only applied to output property names which match the filter. * to apply to all output properties |
+| OutputPropertyName | The name of the output property to assign the input property to |
+| InputPropertyNames | A collection of names of the input property to assign to the output property. Can only specify multiple input properties if the output property is a collection |
+
+## Sample File
+
+``` json
+  "Header": {
+    "InputOntologies": [
+      {
+        "Name": "input-ontology-name",
+        "Version": "1.0",
+        "DtdlVersion": "v2"
+      }
+    ],
+    "OutputOntologies": [
+      {
+        "Name": "output-ontology-name",
+        "Version": "2.0",
+        "DtdlVersion": "v3"
+      }
+    ]
+  },
+  "InterfaceRemaps": [
+    {
+      "InputDtmi": "dtmi:source-namespace:source-interface-name;1",
+      "OutputDtmi": "dtmi:target-namespace:target-interface;1"
+    },
```

Comment:
I'm all for trailing commas, just wanted to make sure that they're supported, and it was intentional that this is here (since its not there on the others)